ffi/diplomat/wasm/lib/api.mjs

```diff
@@ -807,23 +823,23 @@ export class ICU4XPluralOperands {
   }
 
   get v() {
-    return (new Uint32Array(wasm.memory.buffer, this.underlying + 4, 1))[0];
+    return (new Uint32Array(wasm.memory.buffer, this.underlying + 8, 1))[0];
```

Comment:
Are these changes to the offsets correct?

Reply:
OK, so the difference appears to be that ICU4XPluralOperands contains some `u64` values.  Previously, that was considered to be 4 bytes wide, but now it is considered to be 8 bytes wide.

All of the offsets involve ICU4XPluralOperands.